api/src/services/geo-service.ts

```diff
@@ -228,6 +230,13 @@ export class GeoService {
 
   constructor(options?: { baseUrl?: string }) {
     this.baseUrl = options?.baseUrl ?? process.env.BcgwBaseUrl ?? 'https://openmaps.gov.bc.ca/geo/pub/ows';
+
+    axios.interceptors.response.use(
+      (response) => response,
+      (error) => {
+        throw new ApiGeneralError('Failed to geospatial data from external service.', [formatAxiosError(error)]);
```

Comment:
`Failed to retrieve geospatial data from external service`

Reply:
Fixed!